internal/silk/silk.go

```diff
@@ -76,6 +76,15 @@ func clampFloat(low, in, high float32) float32 {
 	return in
 }
 
+func clampUnit(v float32) float32 {
```

Comment:
Where does the term `unit` come from? All for the change, but `clampUnit` would confuse me at first.

I think something more verbose would be one `clampNegativeOneToOne` even.

Reply:
Ah, `unit` is one of the names I've been using myself for denoting -1 to 1 range. I cannot recall where I picked it up -- it could've been something related to graphics programming -- i.e. clamp to a unit vector.

Maybe `clamp11` or just `clamp1` would be easier to read? But, since it's not too commonly used here, `clampNegativeOneToOne` would be fine as well.